.github/workflows/preview.yml

```diff
@@ -0,0 +1,37 @@
+name: Storybook preview
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+      - edited
```

Comment:
This is only added for testing aka trigger deployment if we do changes in this workflow in this PR. IMPORTANT to remove before merging
```suggestion
```